Makefile

```diff
@@ -51,6 +51,9 @@ else
 		ifeq ($(shell uname -m),armv7l)
 			HOST_ARCH = armv7hf
 		endif
+		ifeq ($(shell uname -m),aarch64)
```

Comment:
I think we can support arm64 builds, now that electron supports that

Reply:
Updated!